public/js/clients/firefox/commands.js

```diff
@@ -98,6 +96,16 @@ function getProperties(grip) {
   return objClient.getPrototypeAndProperties();
 }
 
+function prettyPrint(sourceId, indentSize) {
+  const sourceClient = threadClient.source({ actor: sourceId });
+  return sourceClient.prettyPrint(indentSize);
+}
+
+function disablePrettyPrint(sourceId) {
+  const sourceClient = threadClient.source({ actor: sourceId });
+  return sourceClient.disablePrettyPrint();
+}
```

Comment:
I think it's better this way because some protocols may not have the current state. we happen to have it in the current SourceClient, but I bet chrome just has these methods without the current state